modules/sharedaddy/sharing.css

```diff
@@ -564,6 +564,38 @@ body .sd-social-icon .sd-content li.share-custom a span {
 }
 
 
+/**
+ * Screen Reader Text for "Icon Only" option
+ */
+
+.sharing-screen-reader-text {
+	clip: rect(1px, 1px, 1px, 1px);
+	position: absolute !important;
+	height: 1px;
+	width: 1px;
+	overflow: hidden;
```

Comment:
Any reason there isn't a `left: -10000px;` here or similar? I saw this same code recommended here but no "left" in this PR that made it into Jetpack: http://webaim.org/techniques/css/invisiblecontent/